2025-February.html

```diff
@@ -0,0 +1,255 @@
+                                <li>19h30 - 19h35: Opening</li>
+                                <li>19h45 - 20h30: Talk</li>
+                                <li>20h30 - 21h30: Raffle &amp; Social</li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Tonight's Sponsor</h1>
+                    <img id="hoster-large" class="hoster big" src="" alt="" style="min-height: auto; padding-top: 2rem;">
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Tonight's Speaker</h1>
+
+                    <div class="two-columns" style="margin-top: 4em;">
+                        <div>
+                            <img id="speaker-face" class="speaker" src="" alt="">
+                        </div>
+
+                        <div>
+                            <h3 id="speaker-name" class="fancy"></h3>
+                            <p id="speaker-subject" style="margin-top: 2rem"></p>
+                        </div>
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="fancy">Rating</h1>
+
+                    <div class="">
+                        <div>
+                            <img class="meetup" src="img/meetup-large.png" alt="Meetup">
+                            <h2 class="fancy">The Venue</h2>
+                        </div>
+
+                    </div>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Conferences and Events</h1>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Dutch PHP Conference</h1>
+                    <img src="img/conference-logos/dpc-2024.png" alt="">
+                    <h4>March 21st</h4>
+                    <h4>Amsterdam</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Symfony live Paris 2025</h1>
+                    <img src="img/conference-logos/symfony.svg" style="height: 300px;">
+                    <h4>March 27th-28th</h4>
+                    <h4>Paris</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Symfony live Berlin 2025</h1>
+                    <img src="img/conference-logos/symfony.svg" style="height: 300px;">
+                    <h4>April 3rd-4th</h4>
+                    <h4>Berlin</h4>
+                </section>
+
+                <section>
+                    <h1 class="slide fancy">Tonight's Prizes</h1>
+                </section>
+
+                <section>
+                    <h1>Jetbrains Licence</h1>
+                    <img class="hoster big" src="img/hoster-logos/jetbrains.svg" alt="Sponsor: Jetbrains">
+                    <h3 class="fancy">Two to raffle</h3>
```

Comment:
We only have 1. We could raffle 2 last month, because we skipped December. ;)

Reply:
https://github.com/AmsterdamPHP/monthly-slidedeck/pull/60